### PR TITLE
Update to Pilas Engine v2.7.4

### PR DESCRIPTION
**Pull request by manuq**
- Update app to v2.7.4
- Remove hardcoded metadata, now merged upstream.
- Add support for aarch64 architecture.

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is creating a "beta" release on flathub. in case you didn't intend this to be.

**Comment by manuq** [comment]
> This is creating a "beta" release on flathub. in case you didn't intend this to be.

Yeah I wanted to do both: create a beta for testing and a PR to merge after testing the beta. You will be happy to see aarch64 added here now!

File: ar.com.pilas_engine.App.yaml [hunk 1]
```diff
@@ -17,20 +17,13 @@ modules:
...
     buildsystem: simple
     build-commands:
-      - 'install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml'
-      - 'install -Dm644 ${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg'
-      - 'install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
+      - 'install -Dm644 extras/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml'
+      - 'install -Dm644 extras/${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg'
+      - 'install -Dm644 extras/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
     sources:
-      - type: file
```
Comment (hfiguiere) [inline ar.com.pilas_engine.App.yaml:24]: you should remove the files from the repository as well.

File: flathub.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "only-arches": ["x86_64"]
+    "only-arches": ["x86_64", "aarch64"]
```
Comment (hfiguiere) [inline flathub.json:2]: you can just remove the file now.

File: ar.com.pilas_engine.App.yaml (hunk 1 above)
Comment (manuq) [reply ar.com.pilas_engine.App.yaml:24]: Ah good to know. How can I do that? Using `cleanup` in the manifest?

Comment (hfiguiere) [reply ar.com.pilas_engine.App.yaml:24]: `git rm` in the repository and commit

Comment (manuq) [reply ar.com.pilas_engine.App.yaml:24]: ah those files. Of course, thanks.

**Pull request merged**